webui/ocw/lib/db.py

```diff
@@ -70,7 +72,7 @@ def __update_run():
         logger.info('Got {} resources groups from Azure'.format(len(instances)))
         azure.sync_instances_db(instances)
 
-        for region in EC2().list_regions():
+        for region in cfg.getList(['ec2', 'regions'], EC2().list_regions()):
```

Comment:
not 100% sure how it is done in python , but maybe better move getList out of the loop so we will not call `EC2().list_regions()` on every iteration ?